hawc/apps/assessment/constants.py

```diff
@@ -124,3 +125,6 @@ class UncertaintyChoices(models.IntegerChoices):
     THIRTY_THOUSAND = 30_000, "30,000"
     ONE_HUNDRED_THOUSAND = 100_000, "100,000"
     THREE_HUNDRED_THOUSAND = 300_000, "300,000"
+
+
+DSSTOX_SEARCH_VECTOR = SearchVector("dtxsid", "content", config="english")
```

Comment:
we could overwrite the queryset in the Autocomplete search to use this, maybe we should add a card for that